src/git-istage/Application.cs

```diff
@@ -32,54 +34,55 @@ public Application(string repositoryPath, string pathToGit)
+                new ConsoleCommand(ScrollPageUp, ConsoleKey.PageUp, "Selects the line one screen above."),
+                new ConsoleCommand(ScrollPageDown, ConsoleKey.PageDown, "Selects the line one screen below."),
+                new ConsoleCommand(ScrollPageDown, ConsoleKey.Spacebar, "Selects the line one screen below."),
+                new ConsoleCommand(ScrollLeft, ConsoleKey.LeftArrow, ConsoleModifiers.Control, "Scrolls left by one character."),
+                new ConsoleCommand(ScrollRight, ConsoleKey.RightArrow, ConsoleModifiers.Control, "Scrolls right by one character."),
+                new ConsoleCommand(GoPreviousFile, ConsoleKey.LeftArrow, "Go to the previous file."),
+                new ConsoleCommand(GoNextFile, ConsoleKey.RightArrow, "Go to the next file."),
+                new ConsoleCommand(GoPreviousHunk, ConsoleKey.Oem4, "Go to previous change block."),
+                new ConsoleCommand(GoNextHunk, ConsoleKey.Oem6, "Go to next change block."),
+                new ConsoleCommand(Reset, ConsoleKey.R, "When viewing the working copy, removes the selected line from the working copy."),
+                new ConsoleCommand(ResetHunk, ConsoleKey.R, ConsoleModifiers.Shift, "When viewing the working copy, removes the selected block from the working copy."),
+                new ConsoleCommand(Stage, ConsoleKey.S, "When viewing the working copy, stages the selected line."),
+                new ConsoleCommand(StageHunk, ConsoleKey.S, ConsoleModifiers.Shift, "When viewing the working copy, stages the selected block."),
+                new ConsoleCommand(Unstage, ConsoleKey.U, "When viewing the stage, unstages the selected line."),
+                new ConsoleCommand(UnstageHunk, ConsoleKey.U, ConsoleModifiers.Shift, "When viewing the stage, unstages the selected block."),
+                new ConsoleCommand(RemoveLastLineDigit, ConsoleKey.Backspace, "Remove last line digit"),
+                new ConsoleCommand(AppendLineDigit0, ConsoleKey.D0, "Append digit 0 to line."),
+                new ConsoleCommand(AppendLineDigit1, ConsoleKey.D1, "Append digit 1 to line."),
+                new ConsoleCommand(AppendLineDigit2, ConsoleKey.D2, "Append digit 2 to line."),
+                new ConsoleCommand(AppendLineDigit3, ConsoleKey.D3, "Append digit 3 to line."),
+                new ConsoleCommand(AppendLineDigit4, ConsoleKey.D4, "Append digit 4 to line."),
+                new ConsoleCommand(AppendLineDigit5, ConsoleKey.D5, "Append digit 5 to line."),
+                new ConsoleCommand(AppendLineDigit6, ConsoleKey.D6, "Append digit 6 to line."),
+                new ConsoleCommand(AppendLineDigit7, ConsoleKey.D7, "Append digit 7 to line."),
+                new ConsoleCommand(AppendLineDigit8, ConsoleKey.D8, "Append digit 8 to line."),
+                new ConsoleCommand(AppendLineDigit9, ConsoleKey.D9, "Append digit 9 to line."),
+                new ConsoleCommand(ShowHelpPage, ConsoleKey.F1, "Show / hide help page")
```

Comment:
We probably should add `?` as well. It's the canonical shortcut in many tools and websites.